services/api/src/resources/notification/resolvers.ts

```diff
@@ -705,3 +705,23 @@ export const removeAllNotificationsFromAllProjects: ResolverFn = async (
   // TODO: Check rows for success
   return 'success';
 };
+
+export const getAllNotifications: ResolverFn = async (
+  root,
+  args,
+  { sqlClientPool, hasPermission }
+) => {
+  await hasPermission('notification', 'view');
```

Comment:
We likely need a `notification:viewAll` permission created for this that is restricted to platform-owner.